src/__tests__/labs/requests/NewLabRequest.test.tsx

```diff
@@ -48,7 +48,7 @@ describe('New Lab Request', () => {
     const history = createMemoryHistory()
 
     beforeEach(() => {
-      const store = mockStore({ title: '' })
+      const store = mockStore({ title: '', lab: { status: 'loading', error: {} } })
```

Comment:
I really like explicit status in place of isLoading boolean. We have to use this approach in more places.